packages/api/src/resolvers/update/index.ts

```diff
@@ -37,9 +37,10 @@ export const updatePageResolver = authorized<
 
   const pageData = {
     id: input.pageId,
-    title: input.title || undefined,
-    description: input.description || '',
+    title: input.title ?? undefined,
+    description: input.description ?? undefined,
     author: input.byline ?? undefined,
+    savedAt: input.savedAt ? new Date(input.savedAt) : undefined,
```

Comment:
Looks good, if input is invalid it will just throw which i think is what we want.